docs/proc_knative-v050-install-OCP-4x.html

```diff
@@ -155,7 +155,7 @@ <h1>Installing Knative v0.5.0 on an OpenShift cluster using the script provided<
 <i class="fa icon-important" title="Important"></i>
 </td>
 <td class="content">
-Installation requires the OpenShift version <code>0.16.1</code> installer or later. Using the <a href="https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/)">latest version installer</a> is recommended.
+Installation requires the OpenShift version <code>4.1.0-rc.0</code> installer located <a href="https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.1.0-rc.0/)">here.</a> Later versions of the installer are not compatible with the Istio version included with this release.
```

Comment:
There's a trailing `)` in the URL here that will break the link. It was present in the previous link as well it appears, but we might as well fix it now.